Utilities/import-llvm

```diff
@@ -15,41 +15,55 @@ import sys
 import subprocess
 
 # ADT types.
-ADT_imports = ['EpochTracker', 'iterator', 'iterator_range', 'Hashing', 'None', 'Optional',
-               'Statistic', 'StringExtras', 'STLExtras', 'AllocatorList', 'Triple']
+ADT_imports = ['iterator', 'iterator_range', 'APFloat', 'APInt', 'APSInt',
+               'AllocatorList', 'EpochTracker', 'Hashing', 'None', 'Optional',
+               'Statistic', 'StringExtras', 'STLExtras']
 
 # ADT basic data structures.
-ADT_imports += ['APFloat', 'APInt', 'APSInt', 'ArrayRef', 'PointerIntPair', 'SetVector', 'StringRef', 'StringSwitch',
-                'Twine', 'IntrusiveRefCntPtr', 'ilist', 'ilist_base', 'ilist_node', 'ilist_node_base',
-                'ilist_node_options', 'ilist_iterator', 'simple_ilist', 'OptionSet', 'PointerUnion']
+ADT_imports += ['ArrayRef', 'Bit', 'PointerIntPair', 'PointerUnion', 'StringRef',
+                'StringSwitch', 'Triple', 'Twine', 'IntrusiveRefCntPtr',
+                'ilist', 'ilist_base', 'ilist_node', 'ilist_node_base',
+                'ilist_node_options', 'simple_ilist', 'ilist_iterator']
 
 # ADT Mapping structures.
-ADT_imports += ['DenseMap', 'DenseMapInfo', 'DenseSet', 'FoldingSet', 'StringMap', 'StringSet']
+ADT_imports += ['DenseSet', 'DenseMap', 'DenseMapInfo', 'FoldingSet', 'StringMap', 'StringSet']
 
 # ADT "Small" structures.
 ADT_imports += ['SmallPtrSet', 'SmallSet', 'SmallString', 'SmallVector']
 
 # ADT Algorithms.
 ADT_imports += ['edit_distance']
 
+Demangle_imports = ['Demangle', 'ItaniumDemangle', 'MicrosoftDemangle',
```

Comment:
What pulled in the demangler?

Reply:
`Support/Unix/Signals.inc` includes `Demangle.h` it uses it to demangle symbols when printing backtraces.

Reply:
Ah, this was apparently one of the modifications to the files that we warned about at the top of the import script:

    char* d = nullptr;//itaniumDemangle(dlinfo.dli_sname, nullptr, nullptr, &res);

I don't think we want to pull in the demangler.